spirv-headers.pc.in

```diff
@@ -0,0 +1,9 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+
+Name: spirv-headers
```

Comment:
Sorry, I should have caught this before.
Please match the case of the project:  SPIRV-Headers
both here, and in this filename.